v2/emailpassword/nextjs/session-verification/in-ssr.mdx

```diff
@@ -18,66 +18,114 @@ This is applicable for when verifying a session in `getServerSideProps` or `getI
 
 For this guide, we will assume that we want to pass the logged in user's ID as a prop to a protected route. An easier method to achieve this would be to [directly get the user ID from the frontend](../../common-customizations/get-user-info#on-the-frontend), but for this example, we will pass it from the server side.
 
-## 1) We use `getSession` in `getServerSideProps`
+## 1) We parse the `accessToken` JWT in `getServerSideProps`
 
 :::important
 If using `getInitialProps`, the method described below applies as well. The only difference is the way the props are returned (see comments in the code).
 :::
 
 
 ```tsx
-import Session from 'supertokens-node/recipe/session'
-import supertokensNode from 'supertokens-node'
+import jwksClient from "jwks-rsa";
+import JsonWebToken from "jsonwebtoken";
+import type { JwtHeader, JwtPayload, SigningKeyCallback } from "jsonwebtoken";
+import { GetServerSidePropsContext } from 'next';
 // @ts-ignore
-import { backendConfig } from '../config/backendConfig'
-
-export async function getServerSideProps(context: any) {
-
-    // this runs on the backend, so we must call init on supertokens-node SDK
-    supertokensNode.init(backendConfig())
-
-    // this will contain the session object post verification
-    let session
-
-    try {
-        // getSession will do session verification for us
-        session = await Session.getSession(context.req, context.res, {
-        overrideGlobalClaimValidators: () => {
-          // this makes it so that no custom session claims are checked
-          return []
-        }})
-    } catch (err: any) {
-        if (err.type === Session.Error.TRY_REFRESH_TOKEN) {
-
-            // in this case, the session is still valid, only the access token has expired.
-            // The refresh token is not sent to this route as it's tied to the /api/auth/session/refresh API paths.
-            // So we must send a "signal" to the frontend which will then call the 
-            // refresh API and reload the page.
-
-            return { props: { fromSupertokens: 'needs-refresh' } }
-            // or return {fromSupertokens: 'needs-refresh'} in case of getInitialProps
-        } else if (err.type === Session.Error.UNAUTHORISED) {
-            // in this case, there is no session, or it has been revoked on the backend.
-            // either way, sending this response will make the frontend try and refresh
-            // which will fail and redirect the user to the login screen.
-            return { props: { fromSupertokens: 'needs-refresh' } }
-        }
-        
-        throw err
+import { appInfo } from "../config/appInfo";
+
+const client = jwksClient({
+  jwksUri: `${appInfo.apiDomain}${appInfo.apiBasePath}/jwt/jwks.json`,
```

Comment:
do not reply on appInfo here. Users may not have supertokens.init at all in the nextjs ssr backend, so there is no need for them to have appInfo either. Instead, change this to the core uri, and explain that this should point to the core. Use the CoreInjector component we have in our docs. Make this change in all places in your pr

Reply:
done